Extensions/ArtifactEngine/Models/downloadLocation.ts

```diff
@@ -0,0 +1,4 @@
+export enum DownloadLocation {
```

Comment:
I don't think we need an enum for this, since it can either be cache or source, there won't be additional values added later.